ddp-workspace/projects/ddp-dsm-ui/src/app/utils/utils.ts

```diff
@@ -491,7 +491,7 @@ export class Utils {
     }
   }
 
-  public static getPartialDateFormatted(dateString: string, format: string): string {
+  public static getPartialDateFormatted(dateString: string | undefined, format: string): string {
```

Comment:
A bit confusing. If `dateString` is undefined and we don't check for that, how will we call `.split()` method on it? (line 498).

Reply:
Sorry, this was a remnant of a wild goose chase I was on yesterday. It will be removed shortly.